packages/openactive-integration-tests/test/features/cancellation/customer-requested-cancellation/implemented/patch-contains-excessive-properties-error-test.js

```diff
@@ -0,0 +1,84 @@
+/* eslint-disable no-unused-vars */
+const chakram = require('chakram');
+const { RequestState } = require('../../../../helpers/request-state');
+const { FlowHelper } = require('../../../../helpers/flow-helper');
+const { FeatureHelper } = require('../../../../helpers/feature-helper');
+const { GetMatch, C1, C2, B } = require('../../../../shared-behaviours');
+const { itShouldReturnAnOpenBookingError } = require('../../../../shared-behaviours/errors');
+
+const { expect } = chakram;
+/* eslint-enable no-unused-vars */
+
+FeatureHelper.describeFeature(module, {
+  testCategory: 'cancellation',
+  testFeature: 'customer-requested-cancellation',
+  testFeatureImplemented: true,
+  testIdentifier: 'patch-contains-excessive-properties-error',
+  testName: 'Successful booking and unsuccessful cancellation due to PatchContainsExcessivePropertiesError',
+  testDescription: 'PatchContainsExcessivePropertiesError returned because patch request includes other properties than @type, @context, orderProposalStatus and orderCustomerNote',
+  // The primary opportunity criteria to use for the primary OrderItem under test
+  testOpportunityCriteria: 'TestOpportunityBookableCancellable',
```

Comment:
This requires a spec update, as it is not possible to filter for `TestOpportunityBookableCancellable` right now. Approving for now and noting this as a separate issue.